src/Compilers/VisualBasic/Test/Semantic/Compilation/SemanticModelGetDeclaredSymbolAPITests.vb

```diff
@@ -2128,6 +2128,59 @@ End Module
                 Diagnostic(ERRID.ERR_IllegalChar, "$"))
         End Sub
 
+        <WorkItem(59709, "https://github.com/dotnet/roslyn/issues/59709")>
+        <Fact>
+        Public Sub LocalUsing()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlib40(
+<compilation>
+    <file name="a.vb">
+Module M1
+    Class C
+        Sub S() As Object
+            Using writer As System.IO.TextWriter = System.IO.File.CreateText("log.txt")
+                writer.WriteLine("")    'BIND:"writer"
+            End Using
+        End Sub
+    End Class
+End Module
+    </file>
+</compilation>)
+
+            Dim model = GetSemanticModel(compilation, "a.vb")
+            Dim expressionSyntax = CompilationUtils.FindBindingText(Of IdentifierNameSyntax)(compilation, "a.vb", 0)
+            Dim local = DirectCast(model.GetSymbolInfo(expressionSyntax).Symbol, LocalSymbol)
```

Comment:
>LocalSymbol [](http://example.com/codeflow?start=81&length=11)

It looks like we are not testing implementation of added API, but rather behavior of existing internal APIs. This comment is applicable to other tests added in this file.

Reply:
Right, good catch: Fixed: d051862b5c1c04a05a6970076a71f3a389b1d210